### PR TITLE
able to edit redirects

### PR DESCRIPTION
**Pull request by asmega**
Redirects index

image

Changing a redirect

image

### CODE REVIEW

File: app/views/redirects/index.html.erb [hunk 1]
```diff
@@ -49,6 +50,11 @@
             <td><time datetime="<%= redirect.created_at %>" title="<%= redirect.created_at %>"><%= time_ago_in_words(redirect.created_at) %> ago</time></td>
             <td><time datetime="<%= redirect.versions.last.created_at %>" title="<%= redirect.versions.last.created_at %>"><%= time_ago_in_words(redirect.versions.last.created_at) %> ago</time></td>
             <td><%= redirect.versions.last.user.email_local_part %></td>
+            <td>
+              <%= link_to redirect do %>
+                <span class="fa fa-pencil"></span>
```
Comment (jonalport) [inline app/views/redirects/index.html.erb]: Maybe "Edit" here? What if someone can't see pencil?

**Comment by benbarnett** [comment]
Screenshot please! 😃

File: app/views/redirects/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <%= form_for @redirect do |f| %>
+      <div class="form-group">
+        <div class="form__input-wrapper form__input-wrapper--fill-width">
+          <div class="form-group">
+            <div class="form__label-heading">
+              <%= f.label :source %>
+            </div>
+            <div class="form__input-wrapper">
+              <%= f.text_field :source %>
+              <span class="form__hint">Start with a forward-slash</span>
```
Comment (benbarnett) [inline app/views/redirects/show.html.erb]: Looks comical written out for some reason

Comment (asmega) [inline app/views/redirects/show.html.erb]: @jonalport was the copywriter

Comment (jonalport) [inline app/views/redirects/show.html.erb]: My #1 preferred direction of slashing

Comment (jonalport) [inline app/views/redirects/show.html.erb]: Oh I forgot I wrote that - joke's on me! :D

Comment (benbarnett) [inline app/views/redirects/show.html.erb]: Is there validation against alternate slash directions?

Comment (jonalport) [inline app/views/redirects/show.html.erb]: Worth looking into

File: app/views/redirects/index.html.erb (hunk 1 above)
Comment (asmega) [inline app/views/redirects/index.html.erb]: good point

File: app/views/redirects/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            </div>
+            <div class="form__input-wrapper">
+              <%= f.select :redirect_type, Redirect::REDIRECT_TYPES %>
+            </div>
+          </div>
+
+          <div class="nav-article is-fixed">
+            <div class="l-constrained">
+              <div class="nav-article__inner">
+                <span class="nav-article__item nav-article__item--right"></span>
```
Comment (benbarnett) [inline app/views/redirects/show.html.erb]: Wot's this do?

File: app/views/redirects/index.html.erb (hunk 1 above)
Comment (asmega) [inline app/views/redirects/index.html.erb]: done

File: app/views/redirects/show.html.erb (hunk 3 above)
Comment (asmega) [inline app/views/redirects/show.html.erb]: cc @jonalport

File: app/views/redirects/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                  <button class="button--action" name="button" type="submit">
+                    <span class="button__text">Update redirect</span>
+                  </button>
+                </span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    <% end %>
```
Comment (jonalport) [inline app/views/redirects/show.html.erb]: Div-city

Comment (asmega) [inline app/views/redirects/show.html.erb]: more like a conurbation. i just copied it from the new action

File: app/views/redirects/show.html.erb (hunk 3 above)
Comment (jonalport) [inline app/views/redirects/show.html.erb]: Hmm yes, that may be redundant

**Comment by asmega** [comment]
@benbarnett screenshots added

**Comment by benbarnett** [comment]
Looks great. I could see myself in the market for a delete button near those edits one day too :+1:

File: app/views/redirects/show.html.erb (hunk 2 above)
Comment (asmega) [inline app/views/redirects/show.html.erb]: not at the moment

File: app/views/redirects/show.html.erb (hunk 4 above)
Comment (jonalport) [inline app/views/redirects/show.html.erb]: Is that a made-up word?

Comment (asmega) [inline app/views/redirects/show.html.erb]: no. here is a link for the lazy people https://en.wikipedia.org/wiki/Conurbation

**Comment by asmega** [comment]
@benbarnett deletes are on the to do list

File: app/views/redirects/show.html.erb (hunk 4 above)
Comment (jonalport) [inline app/views/redirects/show.html.erb]: Thanks! I have pushed a few style updates.

Comment (asmega) [inline app/views/redirects/show.html.erb]: Thanks @jonalport

**Comment by asmega** [comment]
Any more feedback?

**Comment by munckymagik** [comment]
:+1:

**Pull request merged**